db/warehouse/migrate/20230817154337_remove_tableau_exporter.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveTableauExporter < ActiveRecord::Migration[6.1]
```

Comment:
I thought `cleanup_unused_reports` does this?

Reply:
It does, but we no longer run it on deploy (disabled for other reasons.)  This just enforces that this one is no longer there.